### PR TITLE
[bitnami/argo-workflows] fix config map generation for external database enablement

### PR DESCRIPTION
**Pull request by barbarum**
### Description of the change

Fix config map generation for external database enablement

### Benefits

Currently, when `externalDatabase.enabled` is `true` instead of Postgresql/MySQL subchart is enabled, the invalid configmap is generated. An error is thrown when controller/server apps start as below: 

```bash
time="2022-12-29T08:34:07.993Z" level=fatal msg="error converting YAML to JSON: yaml: line 23: mapping values are not allowed in this context"
```
This is caused by the incorrect template set by `controller.config` while generating database configurations.

### Possible drawbacks

N/A

### Applicable issues

N/A

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/argo-workflows/Chart.yaml [hunk 1]
```diff
@@ -35,4 +35,4 @@ sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/argo-workflow-controller
   - https://github.com/bitnami/containers/tree/main/bitnami/argo-workflow-exec
   - https://argoproj.github.io/workflows/
-version: 5.1.0
+version: 5.1.1
```
Comment (CeliaGMqrz) [inline bitnami/argo-workflows/Chart.yaml:38]:
```suggestion
version: 5.1.2
```

**Review by CeliaGMqrz — COMMENTED** [review]
Thanks for your contribution! Can you update the chart version?

File: bitnami/argo-workflows/Chart.yaml (hunk 1 above)
Comment (barbarum) [reply bitnami/argo-workflows/Chart.yaml]: @CeliaGMqrz Done. Thanks for the review!

**Review by CeliaGMqrz — APPROVED** [review]
Thanks! LGTM

**Pull request merged**